src/test/scala/edu/ie3/simona/agent/grid/ReceivedValuesStoreSpec.scala

```diff
@@ -6,37 +6,35 @@
 
 package edu.ie3.simona.agent.grid
 
-import java.util.UUID
-
-import org.apache.pekko.actor.{ActorRef, ActorSystem}
-import org.apache.pekko.testkit.{TestKit, TestProbe}
-import com.typesafe.config.ConfigFactory
 import edu.ie3.datamodel.graph.SubGridGate
-import edu.ie3.simona.test.common.{TestKitWithShutdown, UnitSpec}
+import edu.ie3.simona.agent.participant.ParticipantAgent.ParticipantMessage
+import edu.ie3.simona.test.common.UnitSpec
 import edu.ie3.simona.test.common.model.grid.SubGridGateMokka
+import org.apache.pekko.actor.testkit.typed.scaladsl.{
+  ScalaTestWithActorTestKit,
+  TestProbe,
+}
+import org.apache.pekko.actor.typed.ActorRef
+
+import java.util.UUID
 
 class ReceivedValuesStoreSpec
-    extends TestKitWithShutdown(
-      ActorSystem(
-        "ReceivedValuesStoreSpec",
-        ConfigFactory
-          .parseString("""
-            |pekko.loggers =["org.apache.pekko.event.slf4j.Slf4jLogger"]
-            |pekko.loglevel="OFF"
-        """.stripMargin),
-      )
-    )
+    extends ScalaTestWithActorTestKit
     with UnitSpec
     with SubGridGateMokka {
 
   // test actorRefs
-  val actorProbe1: TestProbe = TestProbe()
-  val actorProbe2: TestProbe = TestProbe()
-  val actorProbe3: TestProbe = TestProbe()
+  val actorProbe1: TestProbe[ParticipantMessage] =
+    TestProbe[ParticipantMessage]()
+  val actorProbe2: TestProbe[ParticipantMessage] =
+    TestProbe[ParticipantMessage]()
+  val actorProbe3: TestProbe[ParticipantMessage] =
+    TestProbe[ParticipantMessage]()
+  val actorProbe4: TestProbe[GridAgentMessage] = TestProbe[GridAgentMessage]()
 
```

Comment:
Could you give those better names that reflect the actor type they're supposed to mock? E.g. `participant1` etc.